modules/eventing/pages/eventing-examples-high-risk.adoc

```diff
@@ -2,129 +2,671 @@
+
+tail -10 high_risk_txns.log 
+
+2020-03-22T13:20:13.319-07:00 [INFO] "transaction:tx-1521299600-109 *** High Risk Txn: amount: 12506.00 USD exceeds purchase threshold: 12000.00 USD" 
+
+2020-03-22T13:20:13.320-07:00 [INFO] "transaction:tx-1547615650-407 *** High Risk Txn: currency mismatch card: USD != txn: GBP" 
+
+2020-03-22T13:20:13.320-07:00 [INFO] "transaction:tx-1513350267-327 *** High Risk Txn: amount: 53500.00 USD exceeds purchase threshold: 25000.00 USD" 
+
+2020-03-22T13:20:13.323-07:00 [INFO] "transaction:tx-1505402809-074 *** High Risk Txn: currency mismatch card: USD != txn: EUR" 
+
+2020-03-22T13:20:13.332-07:00 [INFO] "transaction:tx-1510327095-039 *** High Risk Txn: amount: 12130.00 USD exceeds purchase threshold:  9500.00 USD" 
+
+2020-03-22T13:20:13.335-07:00 [INFO] "transaction:tx-1514388425-140 *** High Risk Txn: amount: 12506.00 USD exceeds purchase threshold: 12000.00 USD" 
+
+2020-03-22T13:20:13.335-07:00 [INFO] "transaction:tx-1517583691-198 *** High Risk Txn: amount: 12506.00 USD exceeds purchase threshold: 12000.00 USD" 
+
+2020-03-22T13:20:13.338-07:00 [INFO] "transaction:tx-1513263650-200 *** High Risk Txn: amount: 12506.00 USD exceeds purchase threshold: 12000.00 USD" 
+
+2020-03-22T13:20:13.340-07:00 [INFO] "transaction:tx-1514648212-166 *** High Risk Txn: amount: 12506.00 USD exceeds purchase threshold: 12000.00 USD" 
 
+2020-03-22T13:20:13.343-07:00 [INFO] "transaction:tx-1511710690-182 *** High Risk Txn: amount: 12506.00 USD exceeds purchase threshold: 12000.00 USD" 
+----
++
+The default directories for the Eventing Application Logs are as follows:
++
+[cols="20%,80%"]
+|===
+| Platform | Location
 
-. To check the results of the deployed Function, after a sufficient time elapse, access the *Couchbase Web Console* > *Buckets* page
+| Linux
+| /opt/couchbase/var/lib/couchbase/data/@eventing/
 
-. Click _flagged_transactions_ bucket.
-All documents available in this bucket are transactions that are flagged as high-risk transactions.
+| Windows
+| C:\Program Files\Couchbase\Server\var\lib\couchbase\data\@eventing\ +
+(Assumes default installation location)
+
+| Mac OS X
+| /Users/<user>/Library/Application\ Support/Couchbase/var/lib/couchbase/data/@eventing/
+|===
+
+. To check the resulting documents of the deployed Function, access the *Couchbase Web Console* > *Buckets* page. You should see 40 new documents in this bucket.  All documents written to this bucket are transactions that are flagged as high-risk transactions.
++
+image::high_risk_txns_08_bucket.png[,%100]
 +
-image::highrisk_04_buckets.png[,%100]
+** Click the *Documents* link of the *review* bucket.
++
+image::high_risk_txns_08_bucket_documents.png[,%100]
++
+** You can select any document in the *review* bucket, in addition you can select by a specific KEY for example 'transaction:tx-1505315650-403' by cutting and pasting this ID (without quotes) into the *'Document ID'* text box and hitting a return
```

Comment:
```suggestion
** You can select any document in the *review* bucket. In addition you can select by a specific key, for example `transaction:tx-1505315650-403`, by cutting and pasting this ID into the *Document ID* text box and hitting kbd:[Return].
```
Better to use formatting, rather than quotes, to delimit the text to be entered. This means you don't need to remind users not to enter the quotes.